### PR TITLE
[FIX] On-premise GeoIp installation

### PR DESCRIPTION
**Pull request by lse-odoo**
On-premise GeoIP installation documentation added.

OPW-2193581

### CODE REVIEW

**Comment by rim-odoo** [comment]
or is it this one ?

File: website/publish/on-premise_geo-ip-installation.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+===========================================
+Geo IP installation (on-premise database)
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: on-premise**s**

File: website/publish/on-premise_geo-ip-installation.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+============
+
+.. warning:: Please note that the installation depends on your computer operating system and distribution. We will assume here that a Linux operating system is used.
+
+#. Install `geoip2 <https://pypi.org/project/geoip2/>`__ python library
+    .. code-block:: bash
+
+      pip install geoip2
+
+#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`__. You should end up with a file call ``GeoLite2-City.mmdb``
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: call**ed**

File: website/publish/on-premise_geo-ip-installation.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
+===========================================
+Geo IP installation (on-premise database)
+===========================================
+
+Installation
+============
+
+.. warning:: Please note that the installation depends on your computer operating system and distribution. We will assume here that a Linux operating system is used.
+
+#. Install `geoip2 <https://pypi.org/project/geoip2/>`__ python library
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: **P**ython

File: website/publish/on-premise_geo-ip-installation.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+      pip install geoip2
+
+#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`__. You should end up with a file call ``GeoLite2-City.mmdb``
+#. Move the file to the folder ``/usr/share/GeoIP/``
+    .. code-block:: bash
+
+        mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
+
+#. Start the server
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: Restart the Odoo server?

File: website/publish/on-premise_geo-ip-installation.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+.. note:: If for some reason you can't perform step 3 you can use the ``--geoip-db`` option instead.
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: I would rephrase: "If you can't/don't want to locate the geoip database in `/usr/share/GeoIP/`, you can use the --geoip-db option of the Odoo cmd line interface" + add a link to the doc of the CLI

File: website/publish/on-premise_geo-ip-installation.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    This option need the absolute path to the GeoIP database file and will use it as the GeoIP database.
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: s/need/takes

File: website/publish/on-premise_geo-ip-installation.rst [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    For example:
+
+    .. code-block:: bash
+
+        ./odoo-bin --geoip-db= ~/Download/GeoLite2-City.mmdb
+
+.. warning:: In previous Odoo version ``GeoIP`` python library was used. But it is no longer supported with current versions.
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: Unclear.

File: website/publish/on-premise_geo-ip-installation.rst [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+Test
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: How to test GeoIP geolocation in your Odoo website?

File: website/publish/on-premise_geo-ip-installation.rst [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+You should end up with a dictionary indicating the location of the ip address.
+
+.. image:: media/on-premise_geo-ip-installation01.png
+    :align: center
+
+.. note:: If the curly braces are empty ``{}``, it can be for any of the following reason :
+
+   - The browsing IP address is the localhost (``127.0.0.1``) or a local area network one (``192.168.*.*``)
+   - If a reversed proxy is used, make sure to configure it correctly
```
Comment (rim-odoo) [inline website/publish/on-premise_geo-ip-installation.rst]: Add a link to  --proxy-mode doc

File: website/publish/on-premise_geo-ip-installation.rst (hunk 2 above)
Comment (AntoineVDV) [reply website/publish/on-premise_geo-ip-installation.rst]:
```suggestion
#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You should end up with a file call ``GeoLite2-City.mmdb``
```
RST na*ism

**Comment by lse-odoo** [comment]
@rim-odoo done !
